launcher/image/cloudbuild.yaml

```diff
@@ -54,7 +54,4 @@ steps:
 timeout: '3000s'
 
 options:
-  logging: CLOUD_LOGGING_ONLY
   dynamic_substitutions: true
-  pool:
-    name: 'projects/confidential-space-images-dev/locations/us-west1/workerPools/cs-image-build-vpc'
```

Comment:
Remove this private pool requirement, so people can submit the image build in their own project

Reply:
Why did we have this?

Reply:
Used to have this to build images in the private pool. But I realize it's fine to have the image building step without private pool, just put tests in the private pool.

Good thing is people can now use this cloudbuild file directly to build image in their own project for testing and debugging